### ISSUE TITLE
Missiles mid-air after exploding cause damage to structures

### ISSUE THREAD

**Issue by WilliamWeberBerrutti** [OpenHV/OpenHV#928]

**Comment by OcelotWalrus** [comment]
Yes, I noticed that with AA turret buildings: after an aircraft gets shot down, structures below him get some minor damage.

**Comment by GraionDilach** [comment]
Keep in mind that warhead spread is defined in 3D within OpenRA and aircraft flies at 2 cells high in OpenHV.

A lot of game engines "cheat" when it comes to spread-in-air either via having a separate z-spread (either via looking up affected targets only within a cylinder instead of using a sphere) or just ignoring z-data in selected cases altogether (for example, TS/RA2 only calculates spread on the Z axis if the weapon exploded at least half a cell above ground and it also halves the spread), but none of such cheats exist in this engine. This also means that without custom code reimplementing the damaging warheads, it's also not possible to trigger different damagewarhead definitions based on explosion altitude.

**Comment by OcelotWalrus** [comment]
Maybe the OpenHV aircraft z coordinates could be upped a bit to fix that?

**Comment by Mailaender** [comment]
That causes other issues as in then you lose the overview of which cell your aircraft is hovering. It probably also makes the map border interaction even wonkier. That missiles cause some debris or damage when exploding nearby kind of makes sense, or at least is consistent within the game physics. Mines also cause a lot of spread damage, for example.
